components/Account.php

```diff
@@ -287,12 +287,14 @@ public function onRegister()
             }
 
             $rules = (new UserModel)->rules;
+            $names = (new UserModel)->attributeNames;
+            $messages = (new UserModel)->customMessages;
```

Comment:
Change above to:
```php
$model = new UserModel();
```

and then reference `$model->rules`, `$model->attributeNames`, `$model->customMessages` as required.

Reply:
Changes implemented as suggested in 8a09dd279e01a98e72a5468d22c5e693e724947e